### PR TITLE
mac ci make install fix

### PR DESCRIPTION
**Pull request by typhoonzero**

### CODE REVIEW

**Review by gongweibao — DISMISSED** [review]
LGTM

File: paddle/scripts/paddle_build.sh [hunk 1]
```diff
@@ -147,6 +147,7 @@ function cmake_gen() {
         -DINFERENCE_DEMO_INSTALL_DIR=${INFERENCE_DEMO_INSTALL_DIR}
         -DWITH_ANAKIN=${WITH_ANAKIN:-OFF}
         -DPY_VERSION=${PY_VERSION:-2.7}
+        -DCMAKE_INSTALL_PREFIX=${INSTALL_PREFIX:-/paddle/build}
```
Comment (JiabinYang) [inline paddle/scripts/paddle_build.sh:150]: Does this PR fix the problem of "/python/dist/*whl" not found problem when build finish and start build install ?

**Review by JiabinYang — COMMENTED** [review]
Does this PR fix the problem of "/python/dist/*whl" not found problem when build finish and start build install ?

**Review by JiabinYang — APPROVED** [review]
LGTM

File: paddle/scripts/paddle_build.sh (hunk 1 above)
Comment (typhoonzero) [reply paddle/scripts/paddle_build.sh:150]: make install will not have permits when using a normal user other than root

**Pull request merged**